reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/client/JavaClientDriver.java

```diff
@@ -84,6 +84,13 @@ public void runTests() throws IOException {
             }
             line = reader.readLine();
         }
+        // "priming' server, seems to reduce the frequency of random errors
```

Comment:
"priming' probably doesn't need to air quotes...

Also for a TCK, I'd generally argue against fixes that "reduce the frequency of random errors. I would argue you want either 0 random errors or easily reproducible errors.